samples/cellular/modem_shell/src/startup_cmd/startup_cmd_shell.c

```diff
@@ -93,14 +93,20 @@ static int startup_cmd_shell(const struct shell *shell, size_t argc, char **argv
 			startup_cmd_mem_slot = 3;
 			break;
 
-		case '?':
-			mosh_error("Unknown option. See usage:");
-			goto show_usage;
 		case 'h':
```

Comment:
`-h` and `--help` options do not work with `startup_cmd`:

```
mosh:~$ startup_cmd -h
Unknown option. See usage:
...
```

```
mosh:~$ startup_cmd --help
Unknown option. See usage:
...
```

Reply:
Fixed